osu.Game/Tests/Visual/Spectator/TestSpectatorStreamingClient.cs

```diff
@@ -14,7 +15,7 @@ namespace osu.Game.Tests.Visual.Spectator
     public class TestSpectatorStreamingClient : SpectatorStreamingClient
     {
         public new BindableList<int> PlayingUsers => (BindableList<int>)base.PlayingUsers;
-        private readonly HashSet<int> watchingUsers = new HashSet<int>();
+        private readonly ConcurrentDictionary<int, byte> watchingUsers = new ConcurrentDictionary<int, byte>();
```

Comment:
lol what is this. just working around lack of `ConcurrentList`? does `ConcurrentBag` not work here?

Reply:
ConcurrentBag doesn't have a `TryAdd`, which is what I wanted in particular. I can change it to use locking as an alternative.

Reply:
I think it's fine considering it's just a test. Maybe a comment.